docs/articles/expensify-classic/account-settings/Close-Account.md

```diff
@@ -1,5 +1,123 @@
 ---
-title: Close Account
+title: Close Account 
 description: Close Account
----
-## Resource Coming Soon!
+--- 
+# Overview 
+
+Here is a walk through of how to close an Expensify account through the website or mobile app. 
+
+# How to close your account 
+On the Web:
+
+1. Go to **Settings** in the left hand menu. Click **Account**. 
+2. Click “Close Account” 
+3. Follow the prompts to verify your email or phone number.
+4. Your account will be closed, and all data will be deleted.*
+
+![Close Account via Website]({{site.url}}/assets/images/ExpensifyHelp_CloseAccount_Desktop.png){:width="100%"}
+
+On the Mobile App:
+
+Open the app and tap the three horizontal lines in the upper left corner.
+Select Settings from the menu.
+Look for the "Close Account" option in the "Others" section. (If you don’t see this option, you have a Domain Controlled account and will need to ask your Domain Admin to delete your account.) 
+Complete the verification process using your email or phone number.
+Your account will be closed, and all data will be deleted.*
+
+![Close Account on Mobile Application]({{site.url}}/assets/images/ExpensifyHelp_CloseAccount_Mobile.png){:width="100%"}
```

Comment:
this image `ExpensifyHelp_CloseAccount_Mobile` doesn't exist.
Please add it in this PR

<img width="1050" alt="image" src="https://github.com/Expensify/App/assets/29673073/96a36d4a-aad5-4b22-b89d-f81bd539ec17">